src/lib/Scenes/Home/index.tsx

```diff
@@ -123,21 +134,15 @@ export default class Home extends React.Component<Props, State> {
       wider letter-spacing. Going forward, this would ideally be dealt with through letter indentation. */}
           <Tab tabLabel=" Artists">
             <WorksForYouRenderer
-              render={renderWithLoadProgress(query => (
-                <WorksForYou query={query as any} />
-              ))}
```

Comment:
When these functional components were defined inline, they would be new instances on each render. Thus, changing a prop on the query renderer would lead to this entire tree re-rendering, as React has no way of knowing if this is the same component (technically it wasn’t).

Reply:
Gotcha, I was thinking the bug was caused by something like this, but couldn't get it to stick. Thanks Eloy!